### ISSUE TITLE
 The danger of vague, undefined, and misleading wording in the constitution and the need for constitutional consensus.

### ISSUE THREAD

**Issue by honestdude** [EOS-Mainnet/governance#27]
Without constitutional consensus, everyone will interpret it differently, and rightfully so. I have seen very little effort to ensure we are all on the same page regarding the interpretation of the constitution. I am aware effort is being made to rewrite the entire document, still, it is of the utmost importance that we all agree on what those words in the constitution actually mean

First and foremost, we must ensure everyone comprehends the language used, in the same way. We need to clarify the language used in this constitution, define key terms and only then can we really examine the potential impacts of this constitution as it stands. To avoid future conflict and unnecessary disagreement, we should never let the rule (article) define itself.

In what language is this constitution written?

Are we really assuming everyone will comprehend this constitution in the same way? Words mean different things to different people. If we want this to be a universally binding agreement, we need to make certain that everyone comprehends this constitution in the same way. As it reads now, there is too much potential for wiggle room in individual interpretation. Left as is, I would expect to see someone argue in arbitration that they could not and did not, agreed to the constitution, because they never comprehended it in the first place, and thus the articles of the constitution do not apply. 

It seems to be very similar to modern English but I am afraid it could contain some Legalese, the often-misunderstood language of lawyers and judges. I am proudly not a BAR member, the entire language of legalese is incredibly misleading and was likely designed with the intentions of deceiving the native English speakers into actively consenting into obligations contracting under the illusion of comprehension.

One example which illustrates the nature of my concern is the Terms and Conditions which everyone is forced to accept before using a new web application, applying for a new credit/insurance and renting a car. Most folk never read that nonsense, partly because they do not care or will not be bothered by it, and partly because they do not understand it. There is no good reason for this confusion in my mind as the same agreement could easily be written in a language much more palatable to the common man, without the legal cost. Alas, Legalese is not meant to be understood by the common folk. It is written in Legalese.

Personal opinions aside, Black’s Law dictionary is drastically different than Webster’s English dictionary. The language we hear in court is not English. Legalese is an entirely separate language than English, both containing many similar words. I simply want to know what language this constitution is written in. Are any parts of it written in Legalese? I think the constitution itself should clarify which language and the corresponding set of definitions in which should or could be used to interpret this constitutional contract.

This could be written into article VIII for relevancy sake but as it is very important to know the language used before attempting to interpret the previous articles, could “language used” be placed as a preamble or sorts? There are some other definitions I feel like could be more clearly defined as well. They could all be presented at the top of the constitution or even as a glossary if need be. I want to make it absolutely clear what is meant by every word in the constitution.

gloss or Key terms should include: (at least)

Developer- Each Member who makes available a smart contract on this blockchain.

Member

EOS token holder

Arbitration

Termination of agreement- (come on, let's just say after 3 years of inactivity your account will be deleted forever)

Blockchain-

Constitution

Consideration

Perjury

Informed consent

Consent

Choice of Law

Violence

Initiation of violence

Language

Restitution

Open source

Amendment

Constitutional amendment

Riparian contract

Liability

Acceptance

Counterparts

Executed

Delivered

Token

EOS

Token

Vote

After coming to a consensus amongst ourselves about the meaning of the constitutional articles, a strong effort should be made to broadcast this consensus to all EOS users. we could make available a system of "constitutional comprehension support" gratis for all new EOS community members perhaps via a worker proposal.  To make this even more bulletproof, we could write somewhere in the constitution itself an announcement, "do not agree to any contracts, including this constitution, until you fully comprehend the content. If you feel uncertain about any part of this constitution, you are encouraged to reach out to the comprehension support group" be it a telegram, phone line, e-mail, or whatever we should provide the correct contact info in the constitution itself. I ask for help from the community in determining if there could be a way to prevent non-EOS members from spamming the support line? something to prove they actually have EOS? Also should or could the support line be anonymous?

**Comment by villesundell** [comment]
I agree, definitions are needed! I tried to start discussion at #14, it also includes my reasoning why "member" should be defined.
